tests/Helpers/PHPUnit_Framework_TestCase.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Tests\Helpers;
+
+/**
+ * This is helpers class for supporting old and new PHPUnit versions.
+ *
+ * @todo NEXT_MAJOR: Remove this class when dropping support for < PHPUnit 5.4.
```

Comment:
Agree, this is not needed, but I don't know if this was copied from other bundle. If that is the case, this is wrong on a lot of bundles. Let me check:

https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Tests/Helpers/PHPUnit_Framework_TestCase.php

We could remove it here, but it should not matter because we will drop this code soon on next major